docs/sources/k6/next/shared/k6-operator/troubleshooting-common-scenarios.md

```diff
@@ -47,35 +62,19 @@ This case is somewhat similar to the previous two: the k6 Operator will wait ind
 If there's at least one runner Pod that OOM-ed, the whole test will be [stuck](https://github.com/grafana/k6-operator/issues/251) and will have to be deleted manually:
 
 ```bash
-kubectl -f my-test.yaml delete
-# or
 kubectl delete testrun my-test
 ```
 
-In case of OOM, it makes sense to review the k6 script to understand what kind of resource usage this script requires. It may be that the k6 script can be improved to be more performant. Then, set the `spec.runner.resources` in the TestRun CRD, or `spec.resources` in the PrivateLoadZone CRD accordingly.
+A `PrivateLoadZone` test or a `TestRun` [with cloud output](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/k6-operator-to-gck6/#cloud-output) will be aborted by the Grafana Cloud k6 after its expected duration is up.
 
-### PrivateLoadZone: subscription error
-
-If there's an issue with your Grafana Cloud k6 subscription, there will be a 400 error in the logs with the message detailing the problem. For example:
-
-```bash
-"Received error `(400) You have reached the maximum Number of private load zones your organization is allowed to have. Please contact support if you want to create more.`. Message from server ``"
-```
-
-To fix this issue, check your organization settings in Grafana Cloud k6 or contact Support.
-
-### PrivateLoadZone: Wrong token
+#### How to fix
 
-There can be two major problems with the authentication token:
+In the case of OOM, it makes sense to review the k6 script to understand what kind of resource usage this script requires. It may be that the k6 script can be improved to be more performant. Then, set the `spec.runner.resources` in the `TestRun` CRD, or `spec.resources` in the `PrivateLoadZone` CRD accordingly.
 
-1. If the token wasn't created, or was created in a wrong location, the logs will show the following error:
+### Disruption of the k6 runners
 
-   ```bash
-   Failed to load k6 Cloud token	{"namespace": "plz-ns", "name": "my-plz", "reconcileID": "67c8bc73-f45b-4c7f-a9ad-4fd0ffb4d5f6", "name": "token-with-wrong-name", "secretNamespace": "plz-ns", "error": "Secret \"token-with-wrong-name\" not found"}
-   ```
+The k6 test can execute for a long time. But depending on the Kubernetes setup, it may happen that the Pods running k6 are disrupted and moved elsewhere during execution. This will skew the result of the test. In the case of a `PrivateLoadZone` test or a `TestRun` [with cloud output](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/k6-operator-to-gck6/#cloud-output), the test run may additionally be aborted by Grafana Cloud k6 once its expected duration is up, regardless of the exact state of k6 processes.
```

Comment:
```suggestion
A k6 test can be executed for a long time. But depending on the Kubernetes setup, it's possible that the Pods running k6 are disrupted and moved elsewhere during execution. This will skew the test results. In the case of a `PrivateLoadZone` test or a `TestRun` [with cloud output](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/k6-operator-to-gck6/#cloud-output), the test run may additionally be aborted by Grafana Cloud k6 once its expected duration is up, regardless of the exact state of k6 processes.
```